src/Saas.Provider/Saas.Provider.Web/Controllers/CreateController.cs

```diff
@@ -2,17 +2,17 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
-using Saas.Provider.Web.Models;
-using Saas.Provider.Web.Models.CosmosDb;
-using Saas.Provider.Web.Services;
+using Saas.LandingSignup.Web.Models;
```

Comment:
I noticed you changed some of the Provider Files to now reference the Saas.LandingSignup project like here. I think we should either keep the references to the provider app or get rid of the provider app files entirely. I'm leaning towards the latter, unless you think there's any reason we should keep the old provider files around for the time being?